### PR TITLE
Change icon for cirrus-ci-web repository link

### PR DESCRIPTION
**Pull request by RDIL**
Use `fa-code` as this is the source.
The duplicate GitHub icons look ugly together.

### CODE REVIEW

**Comment by fkorotkov** [comment]
I don't think changes in `serve.json` are relevant, aren't they?

**Comment by RDIL** [comment]
No, but react was spitting out warnings because it couldn't load the manifest

File: src/Routes.tsx [hunk 1]
```diff
@@ -256,8 +256,8 @@ class Routes extends React.Component<WithStyles<typeof styles>, { openDrawer: bo
                 target="_blank"
                 rel="noopener noreferrer"
               >
-                <Icon className={classNames('fa', 'fa-github', classes.leftIcon)} />
-                <span className="d-none d-md-block">GitHub</span>
+                <Icon className={classNames('fa', 'fa-code', classes.leftIcon)} />
+                <span className="d-none d-md-block">Source</span>
```
Comment (fkorotkov) [inline src/Routes.tsx]: Let's keep it GitHub. The other icon doesn't have the label just a tooltip.

Comment (RDIL) [reply src/Routes.tsx]: Still, I think this just looks better.

Comment (fkorotkov) [reply src/Routes.tsx]: I don't have strong opinion about it. 🤷‍♂️

Comment (RDIL) [reply src/Routes.tsx]: Thanks for accepting it.

**Pull request merged**